packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -195,6 +204,16 @@ class ScrollPhysics {
   /// Scroll fling velocity magnitudes will be clamped to this value.
   double get maxFlingVelocity => parent?.maxFlingVelocity ?? kMaxFlingVelocity;
 
+  /// A function for carrying scrolling momentum on repeated flings.
+  ///
+  /// The function is applied to the existing scroll velocity when another
+  /// scroll drag is applied in the same direction.
+  ///
+  /// By default, physics for platforms other than iOS doesn't carry momentum.
+  MomentumCarryFunction get momentumCarryFunction {
```

Comment:
I think we can simplify this further if we do it a little like `createBallisticSimulation`, except returning 0.0 instead of null.

That should let you get rid of the `MomentumCarryFunction` type entirely.